README.md

```diff
@@ -36,22 +44,20 @@ To include extra testing dependencies, the syntax is as follows:
 
     pip install "jwst[test] @ git+https://github.com/spacetelescope/jwst"
 
-### Installing a DMS release ###
+### Installing a Data Management System (DMS) release ###
 
-We still package our releases to DMS via environment snapshots that specify the exact versions of all packages to be installed.
+We still package our releases to DMS via environment snapshots that specify the exact versions of all packages to be installed. This method should be more stable than the frist method because the latest versions of packages may cause problems.
```

Comment:
typo: "frist" should be "first"

Reply:
I don't like this wording.  Latest versions of packages should not cause problems, because we test against the latest versions of packages, and generally latest patch versions of packages will have bugfixes, so it should not be an issue.

If you're installing a very old version of the pipeline, then using one of these DMS snapshots *might* be preferable for stability reasons, but is probably not preferred from a performance and bugfix standpoint.  It really depends on the dependency.

Reply:
@jdavies-st I was trying to figure out why you would choose option 2 over option 1. We have a general problem on the project of giving users options but not describing why you would use one option over the other. 
  So, should it say to choose the DMS option only if you are trying to run a non-current version?